docs/source/examples/learning-depolarizing-noise.myst

```diff
@@ -0,0 +1,210 @@
+
+```{code-cell} ipython3
+import numpy as np
+from matplotlib import pyplot as plt
+from cirq import (
+    CXPowGate,
+    DepolarizingChannel,
+    LineQubit,
+    Circuit,
+)
+from mitiq import Executor, Observable, PauliString
+from mitiq.interface.mitiq_cirq import compute_density_matrix
+from mitiq.cdr import generate_training_circuits
+from mitiq.cdr._testing import random_x_z_cnot_circuit
+from mitiq.pec import execute_with_pec
+from mitiq.pec.representations.depolarizing import (
+    represent_operation_with_local_depolarizing_noise,
+)
+from mitiq.pec.representations.learning import (
+    biased_noise_loss_function,
+    learn_depolarizing_noise_parameter,
+)
+```
+
+Since the learning-based workflow uses the function {func}`.cdr.generate_training_circuits` from Mitiq's {ref}`Clifford data regression module
+<guide/cdr/cdr>` to generate the near-Clifford training circuits, the input circuit must be one that is compiled into the Rx-Rz-CNOT gateset. 
+Here we use a simple Rx-Rz-CNOT circuit, with an (optional) seed for reproducibility.
+
+```{code-cell} ipython3
+circuit = random_x_z_cnot_circuit(
+    LineQubit.range(2), n_moments=10, random_state=np.random.RandomState(1)
+)
+print(circuit)
+```
+
+Define the ideal executor function for simulating the Clifford training circuits without noise. 
+This will be used for comparison to the error-mitigated expectation values in the learning function, and for a final comparison with the
+mitigated and unmitigated values at the end of the workflow.
+
+```{code-cell} ipython3
+def ideal_execute(circ: Circuit) -> np.ndarray:
+    # For large circuits, this should be replaced by a near-Clifford simulator.
+    return compute_density_matrix(circ, noise_level=(0.0,))
+
+
+ideal_executor = Executor(ideal_execute)
+```
+
+Define the noisy executor, in this case for simulating depolarizing noise on the `CNOT` gate in each of the training circuits. 
+The optimized value of the noise strength `epsilon` should be close to the value defined for this executor.
+
+```{code-cell} ipython3
+CNOT_ops = list(circuit.findall_operations_with_gate_type(CXPowGate))
+
+operations_to_learn = [Circuit(op[1]) for op in CNOT_ops]
+
+epsilon = 0.05
+
+def noisy_execute(circ: Circuit) -> np.ndarray:
+    noisy_circ = circ.copy()
+    insertions = []
+    for op in CNOT_ops:
+        index = op[0] + 1
+        qubits = op[1].qubits
+        for q in qubits:
+            insertions.append((index, DepolarizingChannel(epsilon)(q))) 
+    noisy_circ.batch_insert(insertions)
+
+    return ideal_execute(noisy_circ)
+
+
+noisy_executor = Executor(noisy_execute)
+```
+
+Before calling the optimizer, let's plot the loss function that will be minimized in the learning routine, over a small range of noise strength values.
+The loss function calls {func}`pec.execute_with_pec`, and we can optionally pass keyword arguments to set the number of PEC samples, among other options. 
```

Comment:
Same problem with the link to API-doc with `execute_with_pec`.

Reply:
Good catch, forgot the `.` before `pec.execute_w_pec`